Code/Framework/AzFramework/Tests/Spawnable/SpawnableEntitiesManagerTests.cpp

```diff
@@ -366,6 +366,24 @@ namespace UnitTest
         }
     }
 
+    TEST_F(SpawnableEntitiesManagerTest, EntitySpawnTicket_Move_Works)
+    {
+        AzFramework::EntitySpawnTicket ticket1(*m_spawnableAsset);
+        AzFramework::EntitySpawnTicket ticket2(*m_spawnableAsset);
+
+        const AzFramework::EntitySpawnTicket::Id ticket1Id = ticket1.GetId();
+        const AzFramework::EntitySpawnTicket::Id ticket2Id = ticket2.GetId();
+
+        AzFramework::EntitySpawnTicket ticketMoveConstructor(AZStd::move(ticket1));
+        EXPECT_TRUE(ticketMoveConstructor.IsValid());
+        EXPECT_TRUE(ticketMoveConstructor.GetId() == ticket1Id);
```

Comment:
nit q: i presume `EXPECT_EQ` isn't available for these types?

Reply:
Good call! I'll change it.